network-api/networkapi/wagtailpages/templates/wagtailpages/publication_page.html

```diff
@@ -32,8 +32,29 @@ <h1 class="h1-heading">
                         {{ page.publication_date }}
                     {% endif %}
 
-                    {% if page.publication_file %}
 
+                    <div class="blog-authors d-flex flex-wrap align-items-center">
```

Comment:
Is this template copy-pasted from elsewhere or is it different? I'm wondering if we have the opportunity to make a template block here.